### PR TITLE
Added html for section 2

### PR DESCRIPTION
**Pull request by MariaBgsl**

### CODE REVIEW

File: index.html
Comment (Evbgsl) [inline index.html:1]: Blank lines should preferably be deleted

File: index.html [hunk 1]
```diff
@@ -33,7 +33,24 @@ <h1 class="download__title">Peaceful shopping mindful monwy</h1>
...
+          ZEN cards have a very neat feature that will help you whenever your electronic devices
+          start failing after the standard warranty ends, which is pretty common.
+        </p>
+        <p class="warranty__copy">
+          When you buy any electronics with your ZEN shopping Mastercard, their warranty gets
+          extended by one year. How does it work? It simply does. Automatically.
+        </p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
```
Comment (Evbgsl) [inline index.html:50]: Use / for close single tag

**Review by Evbgsl — COMMENTED** [review]
Correct and send to the curator

**Review by Evbgsl — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -33,13 +31,23 @@ <h1 class="download__title">Peaceful shopping mindful monwy</h1>
     </div>
   </section>
   <!-- Секция 2-->
-  <section>Секция 2</section>
+  <section class="warranty" id="warranty">
+    <div class="waranty__wraper">
```
Comment (Palych18) [inline index.html:35]: wrapper

File: index.html [hunk 3]
```diff
@@ -33,13 +31,23 @@ <h1 class="download__title">Peaceful shopping mindful monwy</h1>
...
+      <div class="warranty__text-block">
+        <h2 class="warranty__title">1-year warranty boost</h2>
+        <p class="warranty__copy">ZEN cards have a very neat feature that will help you whenever your electronic devices start failing after the standard warranty ends, which is pretty common.</p>
+        <p class="warranty__copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty gets extended by one year. How does it work? It simply does. Automatically.</p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket"/>
```
Comment (Palych18) [inline index.html:42]: Space before /

**Review by Palych18 — COMMENTED** [review]
Fix

**Review by Palych18 — APPROVED** [review]
Good

**Pull request merged**